jest.tz.unit.conf.json

```diff
@@ -3,8 +3,12 @@
     "\\.(css|less)$": "<rootDir>/frontend/test/__mocks__/styleMock.js",
     "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/frontend/test/__mocks__/fileMock.js",
     "^promise-loader\\?global\\!metabase-lib\\/metadata\\/utils\\/ga-metadata$": "<rootDir>/frontend/src/metabase-lib/metadata/utils/ga-metadata.js",
-    "^d3-(.*)$": "<rootDir>/node_modules/d3-$1/dist/d3-$1"
+    "^d3-(.*)$": "<rootDir>/node_modules/d3-$1/dist/d3-$1",
+    "react-markdown": "<rootDir>/node_modules/react-markdown/react-markdown.min.js"
   },
+  "transformIgnorePatterns": [
+    "<rootDir>/node_modules/(?!rehype-external-links/)"
```

Comment:
Need to transpile [`rehype-external-links`](https://github.com/rehypejs/rehype-external-links) since it's only available in ES Module.